ui/shared/components/panel/search/VariantSearchFormPanels.jsx

```diff
@@ -122,6 +122,12 @@ export const annotationFieldLayout = (annotationGroups, hideOther) => fieldCompo
   ) : null,
 ].filter(fields => fields)
 
+const freqFieldLayout = fieldComponents => [
+  <Grid.Row className="field">{[fieldComponents[0], fieldComponents[6]]}</Grid.Row>,
+  <Grid.Row className="field">{[fieldComponents[1], fieldComponents[7]]}</Grid.Row>,
+  ...fieldComponents.slice(2, 6),
```

Comment:
This is a little confusing to read. It looks like it should be putting components 0 and 6 on the top line, then 1 and 7 on the line after that, and then 2-5 on a third row. But based on the screenshot, it looks like 0-5 are on the first row and then 6 and 7 are on the second row, so I'm not sure what happening here. can you restructure this or, if you really can't, add comments to explain how this is expected to render? 

also, in general, if you are rendering lists of components you should add a unique`key` prop to each component in the list. You wouldn't need to do anything to the `...fieldComponents.slice(2, 6)` as they already have keys, but for the new row components you would need to add them. 

also, per my last review, the two "callset" filters should be aligned to the right

Reply:
I have two choices to layout the components into two rows: two rows with multiple components in each row, or one row with multiple elements some of which have two rows.  The first option has difficulties in managing the widths of the components. The second option can utilize the option of `widths="equal"` of the parent component (which is `Form.Group`) to ensure all elements have the same width. Components 0 and 6 are put in the first column and components 1 and 7 in the second column.

Reply:
New layout:
![image](https://user-images.githubusercontent.com/33550059/118985242-49b23180-b94c-11eb-92a5-9fbc0fcf8703.png)

Reply:
I see. So what you actually want to do is put the entire layout inside a Field so it uses the whole width available, and then add nested form groups to control their width and do equal layout
```
<Form.Field>
    <Form.Group widths="equal">
      {fieldComponents.slice(0, MAX_COMPONENTS_PER_ROW)}
    </Form.Group>,
    <Form.Group widths="equal">
      // add empty fields to pad out the second row so the "equal" widths are the same
      {[...Array(fieldComponents.length - MAX_COMPONENTS_PER_ROW)].map(() => <Form.Field />)}
      {fieldComponents.slice(MAX_COMPONENTS_PER_ROW)}
    </Form.Group>
</Form.Field>
```

Reply:
That works but may not be perfect because the parent `Form.Group` in `PanelContent` has another two stuffing `Form.Field` components at both sides of the row. These stuff fields occupy bigger space when the `fieldComponents` are encapsulated into one component.
You can see the result as below:
![image](https://user-images.githubusercontent.com/33550059/119006342-4aec5a00-b95e-11eb-9622-bfa2972a30a0.png)
If you think the size of the stuffing fields is acceptable, I can go with the code you proposed.

Reply:
those fields on the side are there intentionally for padding, the code I proposed I tried locally and looks the way I want it to